Vagrantfile

```diff
@@ -1,11 +1,7 @@
-FLASK_PORT=5000
-POSTGRESQL_PORT=5432
-
 Vagrant.configure("2") do |config|
   config.vm.box = "ubuntu/bionic64"
   config.vm.provision :shell, path: "setup.sh"
-  config.vm.network :forwarded_port, guest: FLASK_PORT, host: FLASK_PORT
- config.vm.network :forwarded_port, guest: POSTGRESQL_PORT, host: POSTGRESQL_PORT
+  config.vm.network "public_network"
```

Comment:
IIUC, this means that the Vagrant machine will get a DHCP IP address when it is started and you will access the app directly from that IP address.
How are you planning on keeping track/publishing of that IP address? Or are you planning on using DYNDNS?

Reply:
When vagrant start up it shows its public IP inside the console. Should I make its public IP static?

Reply:
You really shouldn't be connecting your development VM to the public network. It exposes it to the network outside your development machine which is not something you generally want and can cause issues for other developers down the line because the way the machine works can now be affected by the network the developer's machine happens to be connected to...

Reply:
Oh I see. So making it a private network and set a static IP will be okay?

Reply:
Setting a static IP can also cause issues if it will happen to clash with an IP in someone's home network. The best approach is to not make any hard assumptions and just let agrant allocate an internal dynacmic IP automatically.

Reply:
Ack.

Reply:
@ifireball I changed the network definition to private, executed vagrant up and everything went smooth. But when I executed vagrant destroy and vagrant up again I got this issue https://github.com/hashicorp/vagrant/issues/3083 .
The solution I found that worked for me was https://github.com/hashicorp/vagrant/issues/8878#issuecomment-345112810 .
Should I add this to the Vagrantfile ?

Reply:
Either of those 2 solutions work, but whichever you do, you should have a comment that references the problem and the solution

Reply:
Ok. I think the easiest solution, that also won't require the team to fix it manually every time, would be the Vagrantfile solution.